COVID-19-electronic-health-system/Corona-tracker
# Add minimal customized radiks server

This PR
* adds a folder for the server part; currently, this is just an express server with radiks connecting to mongo db
* is motivated by #108 


**Test plan (required)**
1. start mongo daemon (should listen on the default port 27017)
1. checkout this branch
1. `cd server`
1. `yarn` (or `npm install`)
1. `yarn start`

![Screenshot from 2020-03-20 08-49-56](https://user-images.githubusercontent.com/1449049/77145065-d3841780-6a87-11ea-91ec-f80efdf8dda4.png)
